mkosi/__init__.py

```diff
@@ -174,6 +176,9 @@ def wrapper(*args: Any, **kwargs: Any) -> Dict[T, V]:
     return functools.update_wrapper(wrapper, f)
 
 
+def format_list(seq: Iterator[str]) -> str:
```

Comment:
I appreciate the little saver, but the name is a bit generic.

Reply:
I renamed this to `list_to_string` and added a comment. I you still think it's cryptic, I'm fine with dropping the patch.